docs/1_getting_started.md

```diff
@@ -62,6 +62,12 @@ crawler = Crawler(PublisherCollection.us)
 # crawl one article from a publisher based in the US and print it
 for article in crawler.crawl(max_articles=1):
     print(article)
+
```

Comment:
I would suggest moving this example (lines 65-70) to line 91, so that this code does not conflict with the example output and possible code variation in the lines 73-89

Reply:
That makes sense. Fixed!